### DISCUSSION TITLE
How to include MauiFont, MauiImage in a NuGet package

### DISCUSSION THREAD

**Discussion by Yves-Be** [dotnet/maui#10860] [Q&A] [answered]
With the single project structure, we got build actions like MauiFont and MauiImage to make shared resources possible.
This works great from a class library, but stops working as soon as the library is packaged.

Is there any way to make it possible to include these fonts in a packaged library?

I tried setting pack to true, but this does not have any effect.
`    <MauiFont Include="Resources\Fonts\icons.ttf" Pack="true"/>`

I also tried adding the font to Platform/Android/Resources/font/, but this also no longer works. The font is not included in the nuget package.

The only solution seems to be to step down from the single project solution, and move back to a separate project per platform, combined with a nuspec file to be able to publish the library as a single nuget package.

**Comment by Yves-Be** [comment, marked as answer]
For now, it seems that adding the font as embedded resource, and adding an ExportFont attibute to the assembly is the only solution to use fonts from inside a nuget package.
`[assembly: ExportFont("icons.ttf", Alias = "icons")]`

**Reply by Take-A-Byte** [reply]
What about images and other assets?

**Reply by Xyncgas** [reply]
you can set them as embedded resource
